### ISSUE TITLE
Glance

### ISSUE THREAD

**Issue by samuelmeuli** [sindresorhus/quick-look-plugins#121]
Hi @sindresorhus,

I just released [Glance](https://github.com/samuelmeuli/glance), an all-in-one Quick Look plugin. I think it would make a great addition to this list and could even replace some of the existing entries.

**Advantages:**

- Most users will probably only need the file types covered by Glance, so a single app would be enough for them
- Consistent UI across file types
- Dark Mode support
- Uses the new Quick Look API:
  - Doesn't require users to [change any quarantine attributes](https://github.com/sindresorhus/quick-look-plugins#catalina-notes)
  - Some of the other plugins still use the deprecated Quick Look Generator API and might stop working in the future (see https://github.com/sindresorhus/quick-look-plugins/issues/115#issuecomment-546612663)
- Available on the App Store, so it's sandboxed and reviewed

Would you be open for a PR to include my app in the list?

**Comment by Jahhein** [comment]
This is awesome!

I think it would be an awesome addition too! This will make installing and keeping track of all the individual package installs much easier!

You should edit your PR and add a link to the macOS App Store too.

**Comment by texastoland** [comment]
For documentation this replaces:

- QLColorCode (without Lua and Highlight dependencies)
- QLStephen
- QLMarkdown ([except for images](https://github.com/samuelmeuli/glance#user-content-faq))
- BetterZipQL

But not:

- QuickLookJSON (samuelmeuli/glance#14)
- Suspicious Package (samuelmeuli/glance#8 and not working on Catalina anyway)

The others seem out of scope and only QLVideo is supported on Catalina.

**Comment by ladislas** [comment]
Glance has been recently archived and Big Sur is not supported.

@samuelmeuli why not create an org to have the community keep working on it? your readme doesn't say anything about the archival.

**Comment by crunchtime-ali** [comment]
> Glance has been recently archived and Big Sur is not supported.
> 
> @samuelmeuli why not create an org to have the community keep working on it? your readme doesn't say anything about the archival.

Samuel has become a Graduate Software Engineer at Apple in June 2020. I don't see how that would prevent him from continuing the project but maybe he can chime in here.

**Comment by sindresorhus** [comment]
Apple employees are not allowed to have apps on the App Store.

**Comment by ladislas** [comment]
> Apple employees are not allowed to have apps on the App Store.

Makes sense! But a simple note in the readme would prevent any frustration :)

**Comment by texastoland** [comment]
That's a bummer! I guess keep an eye on the [dependency graph](https://github.com/samuelmeuli/glance/network) in case someone else picks it up 🤞🏼

**Comment by kopischke** [comment]
It’s not open source, but there is a _Glance_ alternative on the Mac App Store: [_iPreview_](https://apps.apple.com/de/app/ipreview-powerful-quick-look/id1519213509?mt=12). I’m not affiliated and YMMV, but it works well enough for me.

**Comment by yous** [comment]
FWIW, there is another option: https://github.com/sbarex/SourceCodeSyntaxHighlight

**Comment by Jax0rz** [comment]
I can't found Glance on the AppStore,but I found [iPreview](https://findergg.com) which as same as Glance.

**Comment by crunchtime-ali** [comment]
> I can't found Glance on the AppStore,but I found [iPreview](https://findergg.com) which as same as Glance.

The author likely had to pull Glance from the AppStore because he works for Apple now.

**Comment by chrismessina** [comment]
> 
> The author likely had to pull Glance from the AppStore because he works for Apple now.

Since Apple employees can't work on open source projects (except if they're given special considerations) we can assume this is true.

Therefore — is anyone here willing to fork his repo and resurrect the project?

https://github.com/samuelmeuli/glance

**Comment by chamburr** [comment]
Hi, I recently came across Glance and created a fork of it [here](https://github.com/chamburr/glance). It is now updated to support macOS Big Sur, and I would likely be maintaining it for the future. It would be great to include the app in the list.

Also, I'm currently having some issues with app distribution, due to the Apple Developer Account age requirement. The users have to sign the app themselves and that's very bad. If anyone knows of an alternative or could help with publishing the app, please let me know! Thanks.

**Comment by samtsevich** [comment]
Seems like this app https://github.com/chamburr/glance should be added to the main README.md file :)
